src/NHSD.BuyingCatalogue.Solutions.Application/Persistence/Epics/SolutionEpicsUpdater.cs

```diff
@@ -0,0 +1,24 @@
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading;
+using System.Threading.Tasks;
+using NHSD.BuyingCatalogue.Solutions.Application.Domain.Epics;
+using NHSD.BuyingCatalogue.Solutions.Contracts.Persistence;
+
+namespace NHSD.BuyingCatalogue.Solutions.Application.Persistence.Epics
+{
+    internal sealed class SolutionEpicsUpdater
+    {
+        private readonly ISolutionEpicsRepository _solutionEpicsRepository;
```

Comment:
Could we change ISolutionEpicsRepository interface (& class) to ISolutionEpicRepository, all the rest of the repositories are singular.